### PR TITLE
Newnerdbingo

### LINKED ISSUE #56: Card idea: news nerd bingo
Saw this [on Twitter](https://twitter.com/bizweekgraphics/status/1116385470534045696/photo/1). Could be implemented as a card here!

### PR DESCRIPTION
**Pull request by amysheep**
Closes #56 

~closing out #56~

### CODE REVIEW

**Comment by amysheep** [comment]
@jennybc ready to merge.

File: inst/topics/news-nerd.R [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "RTs any map with hillshades",
+  "worships Amanda Cox",
+  "d3 v4 4ever",
+  "some personal news",
+  "googling flexbox",
+  "srccon community call",
+  "relies on a guy named kyng chaos",
+  "Census update dates in calendar",
+  "very strong opinions about the Data Visualization Society",
+  '\"we are not a service desk\"',
```
Comment (jennybc) [inline inst/topics/news-nerd.R]: Nitpick: the quoting approach here doesn't match elsewhere.

File: inst/topics/news-nerd.R [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "Havre de Grace",
+  "writes?",
+  "has googled \"how to center div\" 300 times",
+  "has had canvas on resum for three years but never used it",
```
Comment (jennybc) [inline inst/topics/news-nerd.R]: I suspect this is meant to be "resume"?

File: tests/testthat/test-package.R [hunk 3]
```diff
@@ -5,6 +5,7 @@ test_that("duplicate squares are removed", {
   expect_identical(c("a", "b"), vet_squares(x))
 })
 
+
```
Comment (jennybc) [inline tests/testthat/test-package.R]: Is this an intentional change in this PR?

**Comment by jennybc** [comment]
Thanks!

In the future, if you use the [magic keywords](https://help.github.com/en/articles/closing-issues-using-keywords) in a commit or in the PR description (not title), merging the PR will close the issue. I'll take care of it in this case.

**Pull request merged**